MANIFEST.in

```diff
@@ -1,10 +1,4 @@
 include README.md
 include LICENSE
-include requirements.txt
-include pybert/images/icon.png
-recursive-include docs/
-recursive-include tests/
```

Comment:
My understanding which might be wrong, when a user downloads your wheel or conda package that it would include the tests folder but none of the other package infrastructure like our pre-commit file or other.  So if you want to develop, we want to force them to clone this repo.  That is also less files/space in the download for install time? (Again, I could be completely wrong.)

Reply:
Okay, makes sense; thanks!